Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin(jenkinsVersions: [null, '2.60'])
+buildPlugin(jenkinsVersions: [null, '2.143'])
```

Comment:
We prefer to test against LTS versions or their baselines. Thus, 2.138.1 currently. Or just nothing

```groovy
buildPlugin()
```

when the `jenkins.version` is already on the current LTS.